docs/telemetry.md

```diff
@@ -0,0 +1,44 @@
+# Telemetry
+
+The Telemetry function enables the Operator gathering and sending basic anonymous data to Percona, which helps us to determine where to focus the development and what is the uptake for each release of Operator.
+
+The following information is gathered:
+
+* ID of the Custom Resource (the `metadata.uid` field)
+* Kubernetes version
+* Platform (is it Kubernetes or Openshift)
+* PMM Version
+* Operator version
+* Mongo version
+* Percona Backup for MongoDB (PBM) version
+* Is sharding enabled (starting from the Operator version 1.13)
+* Is Hashicorp Vault enabled (starting from the Operator version 1.13)
+* Is Operator deployed in a cluster-wide mode (starting from the Operator version 1.13)
+
+We do not gather anything that identify a system, but the following thing should be mentioned:
+Custom Resource ID is a unique ID generated by Kubernetes for each Custom Resource.
+
+Telemetry is enabled by default and is sent to the [Version Service server](update.md#automatic-upgrade) when the Operator connects to it at scheduled times to obtain fresh information about version numbers and valid image paths needed for the upgrade.
+
+The landing page for this service, [check.percona.com](https://check.percona.com/), explains what this service is.
+
+You can disable telemetry with a special option when installing the Operator:
+
+* if you [install the Operator with helm](helm.md), use the following installation command:
+
+  ```sh
+  $ helm install my-db percona/psmdb-db --version {{ release }} --namespace my-namespace --set disable_telemetry="true"
+  ```
+
+* if you don't use helm for installation, you have to edit the `operator.yaml`
+  before applying it with the `kubectl apply -f deploy/operator.yaml` command.
+  Open the `operator.yaml` file with your text editor, find the
+  `disable_telemetry` key and set it to `true`:
+
+  ```yaml
+  ...
+  disable_telemetry: "true"
+  ...
+  ```
```

Comment:
this is only valid for PG operator, users need to update https://github.com/percona/percona-server-mongodb-operator/blob/main/deploy/operator.yaml#L45-L46